src/content/docs/reference/membrane-module.md

```diff
@@ -0,0 +1,255 @@
+---
+title: membrane module
+---
+
+Membrane programs can import functions and types from the `"membrane"` module.
```

Comment:
I think it'd be helpful to mention what the purpose of this module is. Maybe something along these lines?

```suggestion
Programs can access most of Membrane's functionality by importing the `"membrane"` module. It provides exports that can be used to store program state, communicate with other programs and more.
```

Reply:
Also, I think it would be helpful to add that this module is updated every time the schema or the program connections change.